### PR TITLE
fix: iterate through repository language nodes

### PR DESCRIPTION
**Pull request by RayhanADev**
Just made a little oopsie I forgot to actually count the language after getting more from the API. Iterating (`.forEach(...)`) through each repository's language nodes should fix the problem.

### CODE REVIEW

**Comment by RayhanADev** [comment]
@ryo-ma before you merge this, if you wouldn't mind proofreading my Typescript for errors that would be much appreciated :).

**Review by ryo-ma — APPROVED** [review]

**Comment by ryo-ma** [comment]
Ok. Good catch. 
I found the syntax error in this pull request. I will fix it.

**Comment by ryo-ma** [comment]
Done.

**Comment by RayhanADev** [comment]
Did you mean to close this pr without a merge?

**Comment by RayhanADev** [comment]
Wait it is merged, huh that's weird.

Oh well Github is strange ƪ(˘⌣˘)ʃ. Thank's for catching that error ;).

**Pull request closed without merging**
